doc/dev/testing/end-to-end.md

```diff
@@ -1,50 +1,60 @@
 # E2E Testing the Operator SDK
+
 The operator-sdk is automatically tested using both unit tests and e2e tests anytime
 a pull request is made. The e2e tests ensure that the operator-sdk acts as intended by
 simulating how a typical user might use the SDK. The automated tests test each PR and run in
 Travis CI, and Travis CI has a couple of features to simplify the e2e tests that we run.
 However, it is possible to run the e2e tests locally as well.
 
 ## Running the E2E Tests Locally
+
 To run the tests locally, the tests either need access to a remote kubernetes cluster or a
 local minikube instance running on the machine.
 
 ### Remote Kubernetes Instance
+
 To run the tests on a remote cluster, the tests need access to a remote kubernetes cluster
 running kubernetes 1.10 as well as a docker image repo to push the operator image to,
 such as quay.io. To run the test, use this command:
-```
-$ go test ./test/e2e/... -kubeconfig "path-to-config" -image "<repository>:<tag>"
+
+```sh
+$ make test/e2e/go ARGS="-kubeconfig 'path-to-config' -image '<repository>:<tag>'"
 ```
 
 This will run the tests on the cluster specified by the provided kubeconfig and the
 memcached-operator image that is built will be pushed to `<repository>:<tag>`.
 
 ### Local Minikube Instance
+
```

Comment:
Yes. There is nothing in the tests that prevent the tests from running on minikube, we just don't do it on travis because setting it up is more complicated and the openshift cluster is really just a more restrictive kubernetes cluster with extra features, so that's good enough.